libraries/api-extractor/src/DocElementParser.ts

```diff
@@ -159,7 +160,8 @@ export default class DocElementParser {
 
       // Make sure only a single url is given
       if (urlContent.length > 1 && urlContent[1] !== '' ) {
-        documentation.reportError('Invalid @link parameter, url must be a single string.');
+        documentation.reportError('The {@link} tag contains additional spaces after the URL;'
```

Comment:
I made it URL everywhere

---
In reply to: [122554191](https://github.com/Microsoft/web-build-tools/pull/239#discussion_r122554191) [](ancestors = 122554191)